server/v2/controllers/signinController.js

```diff
@@ -0,0 +1,17 @@
+import authenticationCheck from '../middleware/authenticationCheck';
+import UserModel from '../models/userModel';
+
+import Response from '../helpers/responseHelper';
+
+async function signin(body, res) {
+  const { email, password } = body;
+  const user = await UserModel.signin(email, password);
+  if (user) {
+    return authenticationCheck.signNewToken(user, res, 200);
```

Comment:
@Cheza-Dzabala , are you encrypting all user data here? if yes, that is a bad practice. you should only encrypt few data that will be useful on the front end of things (i.e for making a return request)

Reply:
I have a middleware that encrypts the data. I am sending that middleware the user object, the response and the appropriate status code to return on success.